docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -258,8 +280,41 @@ metrics. The default is `127.0.0.1`. The metrics server respects the [`--host-wh
 metrics-port="6174"
 ```
 
-Specifies the port on which the [Prometheus](https://prometheus.io/) monitoring service accesses Pantheon
-metrics. The default is `9545`. The metrics server respects the [`--host-whitelist` option](#host-whitelist).
+Specifies the port on which [Prometheus](https://prometheus.io/) accesses [Pantheon metrics](../Using-Pantheon/Debugging.md#monitor-node-performance-using-prometheus) 
+or which Pantheon pushes metrics to a [Prometheus Pushgateway](https://github.com/prometheus/pushgateway).
+The default is `9545`. 
+
+### metrics-prometheus-job
```

Comment:
This changed to `--metrics-push-prometheus-job` as it only applies to push gateway support.  otherwise all still correct.

I would recommend having this after the host and port options for push.

Reply:
Done